src/zinoargus/__init__.py

```diff
@@ -277,6 +278,47 @@ def synchronize_continuously(argus_incidents: IncidentMap, zino_cases: CaseMap):
             synchronize_case_history(case, incident)
 
 
+def refresh_argus_incidents(argus_incidents: IncidentMap, zino_cases: CaseMap):
+    """Refreshes all the Argus incidents that we know of and care about.
+
+    This also triggers actions that modify Zino cases if indicated by changes to the
+    Argus incidents.
+    """
+    for case_id, old_incident in argus_incidents.items():
+        new_incident = _argus.get_incident(old_incident.pk)
+        argus_incidents[case_id] = new_incident
+
+        if new_incident.acked and not old_incident.acked:
+            update_case_acknowledged(case_id, argus_incidents, zino_cases)
+
+
+def update_case_acknowledged(
+    case_id: int, argus_incidents: IncidentMap, zino_cases: CaseMap
+):
```

Comment:
```suggestion
def update_case_acknowledged(
    case_id: int, argus_incident: Incident, zino_case: ritz.Case
):
```

Since we only work with one Incident/Case it doesn't make much sense to me sending the whole maps around